README.md

```diff
@@ -1138,6 +1138,12 @@ The `jumpstation` connection options support the same values (for as much as it
 
 <a name="release_history"></a>
 # Release History
+* Overthere 5.0.21 (07-Oct-2020)
+    * Upgrade to SMBJ 0.10.0 and Make artifact-copy-strategy work on satellite
```

Comment:
Version changes are cumulative. That means we dont have to mention " and Make artifact-copy-strategy work on satellite" on this line.